nautobot/dcim/forms.py

```diff
@@ -307,6 +307,19 @@ class LocationTypeFilterForm(NautobotFilterForm):
     content_types = MultipleContentTypeField(feature="locations", choices_as_strings=True, required=False)
 
 
+class LocationTypeBulkEditForm(TagsBulkEditFormMixin, NautobotBulkEditForm):
+    pk = forms.ModelMultipleChoiceField(queryset=LocationType.objects.all(), widget=forms.MultipleHiddenInput())
+    description = forms.CharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
+    nestable = forms.NullBooleanField(
+        required=False, widget=BulkEditNullBooleanSelect, label="Allow parents/children ?"
```

Comment:
This is a different `label` than `nestable` uses elsewhere, and is a bit inaccurate/confusing. I'd suggest removing it.